### PR TITLE
Document command for Homebrew plugin install

### PR DESCRIPTION
**Pull request by ivy**
This is a small change to the Homebrew install docs to recommend a one-liner instead of the manual approach:

```
$(brew --prefix certbot)/libexec/bin/pip install certbot-s3front
```

### CODE REVIEW

**Comment by ivy** [comment]
@dlapiduz Just a friendly reminder to have a look at this PR when you have some time. Let me know if you need more info or if something needs to change. 😄

File: README.md [hunk 1]
```diff
@@ -26,14 +26,13 @@ The easiest way to install both the certbot client and the certbot-s3front plugi
...
+  Installed certbot using Homebrew on Mac (as the official way to install on a Mac)? Install the plugin with:
 
   ```bash
-  cat $(which certbot) | head -1
+  $(brew --prefix certbot)/libexec/bin/pip install certbot-s3front
   ```
 
-  Then use the full path to the `pip` binary found in the same folder to install certbot-s3front.
-  Note, you will need to re-install the plugin each time Homebrew will update certbot
+  **Note:** You will need to re-install the plugin each time certbot is updated through Homebrew.
```
Comment (dlapiduz) [inline README.md]: Did you mean to say "...each time _python_ is updated..."?

**Comment by dlapiduz** [comment]
Thanks for the PR. I added a quick comment, once that is clear we can merge it.

File: README.md (hunk 1 above)
Comment (ivy) [reply README.md]: As far as I can tell, the plugin only needs a reinstall whenever certbot itself is updated. The Homebrew formula uses virtualenv to vendor its dependencies under `/usr/local/Cellar/certbot/<version>`.

**Pull request merged**